framework/tlvf/AutoGenerated/include/tlvf/ieee_1905_1/tlvVendorSpecific.h

```diff
@@ -39,8 +40,11 @@ class tlvVendorSpecific : public BaseClass
         };
         
         const eTlvType& type();
-        uint16_t& length();
+        const uint16_t& length();
```

Comment:
What is the point of passing the length as const reference? We pass const reference only if the size is greater than 4 bytes. which is not the case here. (nitpick)

https://github.com/prplfoundation/prplMesh/blob/master/CODINGSTYLE.md#functions

Reply:
This is autogenerated code, this means that the length field should never change. We only pass references in tlvf, so const is needed here.